src/ui/ReactionButton/__tests__/__snapshots__/ReactionButton.spec.js.snap

```diff
@@ -8,8 +8,8 @@ exports[`ReactionButton should do a snapshot test of the ReactionButton DOM 1`]
   role="button"
   style={
     Object {
-      "height": "34px",
-      "width": "34px",
+      "height": "36px",
+      "width": "36px",
```

Comment:
Lets check this? default have changed from 34 to 36

Reply:
cc @HoonBaek

Reply:
<img width="459" alt="image" src="https://user-images.githubusercontent.com/46333979/186795224-edad6289-33b2-49c9-87c8-012217ad7542.png">
Yeah, the expected reaction button size is 36px